src/main/scala/tech/cryptonomic/conseil/generic/chain/DataTypes.scala

```diff
@@ -1,5 +1,7 @@
 package tech.cryptonomic.conseil.generic.chain
 
+import java.io.Serializable
```

Comment:
no need, there's an alias in `_root_.scala` package

Reply:
I don't know why intellij did not see `Serializable` at that moment but you are right, there it is :)